### ISSUE TITLE
Evaluate study: "Bitcoin Laundering: An Analysis of Illicit Flows into Digital Currency Services"

### ISSUE THREAD

**Issue by nopara73** [nopara73/ZeroLink#63]
https://info.elliptic.co/whitepaper-fdd-bitcoin-laundering

Related: https://github.com/nopara73/ZeroLink/issues/62

**Comment by nopara73** [comment]
> However, what we know about Bitcoin’s illicit use is mainly based on anecdotal evidence, usually without supporting data, analysis of how it is used across geographical regions, or trends over time. 

Seems like nobody bothered to study it so far.

---

> Through extensive analysis of a narrow data sample of Bitcoin transactions between 2013 and 2016, this study identifies trends in the flow of bitcoins from clearly identified illicit activity to various digital currency conversion services.  
> The amount of observed Bitcoin laundering was small (less than one percent of all transactions entering conversion services), but what is most relevant are the clear patterns we discovered relating to how illicit bitcoins are laundered. 
> This study considers well over half a million bitcoins that moved directly from illicit sources to conversion services in the period 2013-2016.

1. They confuse Bitcoin mixing with Bitcoin laundering.
2. Less than 1% of Bitcoin transactions were mixing.  

Combining this data with the estimated USD value: https://blockchain.info/charts/estimated-transaction-volume-usd?timespan=all&daysAverageString=7  

Where we would get roughly average 50 million USD mixed per week between 2013 to 2016.

It's also interesting to note that in 2016 the average value transacted per week was about 100 million USD, and in 2017 around 1 billion USD.

---

> We found that darknet marketplaces such as Silk Road and, later, AlphaBay, were the source of almost all of the illicit bitcoins laundered through conversion services that we identify in our study. Bitcoin exchanges received the greatest amount of identified illicit bitcoins out of all conversion services, but they also processed the majority of Bitcoin transactions overall.

The direction of money flow was from marketplaces to mixers to exchanges.

---

Europe was mixing the most, then North America, then everything in between, and a little bit Asia was mixing, too.

---

In conclusion, this document, while interesting, does not have anything to do with ZeroLink.
